pyanaconda/modules/payloads/payload/rpm_ostree/installation.py

```diff
@@ -86,6 +95,10 @@ def _get_stateroot(data):
     :param data: OSTree source structure
     :return str: stateroot or osname value based on source
     """
+    if isinstance(data, BootcConfigurationData):
```

Comment:
This is not needed as is_container should be also true and the condition bellow will be satisfied?

Reply:
In the past we added `isBootc` function to the base class but it was removed after one of reviews and replaced with `isinstance` call instead.